src/expandable/multi_iter_request.rs

```diff
@@ -0,0 +1,37 @@
+use yaml_rust::Yaml;
+
+use actions::{Request, Runnable};
+
+pub fn is_that_you(item: &Yaml) -> bool{
+  item["request"].as_hash().is_some() &&
+  item["with_items_iter"].as_hash().is_some()
+}
+
+pub fn expand(item: &Yaml, list: &mut Vec<Box<(Runnable + Sync + Send)>>) {
+  let with_items_iter_option = item["with_items_iter"].as_hash();
+
+  if with_items_iter_option.is_some() {
+    let with_iter_items = with_items_iter_option.unwrap();
+
+    let mut start = 1i64;
```

Comment:
I think we can avoid those `mut` variable with `if let` + `unwrap_or` clauses. Something like:

```
let start = with_iter_items.get(&ystart).unwrap().as_i64().unwrap_or(1i64);
```

Reply:
Perfect!